includes/ucf-post-list-shortcode.php

```diff
@@ -17,11 +17,13 @@
 	function sc_ucf_post_list( $atts, $content='' ) {
 		$atts = shortcode_atts( UCF_Post_List_Config::get_shortcode_atts(), $atts, 'ucf-post-list' );
 		$layout = isset( $atts['layout'] ) ? $atts['layout'] : 'default';
+		$show_image = isset( $atts['show_image'] ) ? filter_var( $atts['show_image'], FILTER_VALIDATE_BOOLEAN) : true;
+		$posts_per_row = isset( $atts['posts_per_row'] ) ? $atts['posts_per_row'] : 0;
```

Comment:
Once you add `format_callback` functions for `show_image` and `posts_per_row`, you shouldn't need to apply `filter_var()` to `$show_image` here.

Also by doing that, you should be able to pass `$atts['show_image']` and `$atts['posts_per_row']` directly to `UCF_Post_List_Common::display_post_list()`.